invenio_rdm_records/services/config.py

```diff
@@ -827,8 +847,8 @@ class RDMCommunityRecordsConfig(BaseRecordServiceConfig, ConfiguratorMixin):
         "RDM_SEARCH",
         "RDM_SORT_OPTIONS",
         "RDM_FACETS",
-        search_option_cls=RDMSearchOptions,
-        search_option_cls_key="RDM_SEARCH_OPTIONS_CLS",
+        search_option_cls=RDMCommunityRecordSearchOptions,
+        search_option_cls_key="RDM_COMMUNITY_RECORD_SEARCH_OPTIONS_CLS",
```

Comment:
comment: I was about to say to add this to the top-level config.py of the module, but I see the previous config isn't there either... So LGTM :)